src/Services/Air/AirParser.js

```diff
@@ -1084,6 +1084,29 @@ function universalRecordRetrieveRequest(data) {
   return response;
 }
 
+function formatProviderReservationInfo(data) {
+  return {
+    traveler: data['common_v47_0:BookingTravelerName'].map((travelerDetails) => {
```

Comment:
Would prefer to call it passengers, as we never used `traveler` before and it is array.